content/en/blog/_posts/2020-04-22-Supporting-the-Evolving-Ingress-Specification-in-Kubernetes-1.18.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: blog
+title: Supporting the Evolving Ingress Specification in Kubernetes 1.18
+date: 2020-04-22
+slug: Supporting-the-Evolving-Ingress-Specification-in-Kubernetes-1.18
+---
+
+**Authors:** Alex Gervais (Datawire.io)
+
+Last month, the Kubernetes team released [Kubernetes 1.18](https://kubernetes.io/blog/2020/03/25/kubernetes-1-18-release-announcement/), which extended Ingress. In this blog post, we’ll walk through what’s new in the new Ingress specification, what it means for your applications, and how to upgrade to an ingress controller that supports this new specification.
+
+### What is Kubernetes Ingress
+When deploying your applications in Kubernetes, one of the first challenges many people encounter is how to get traffic into their cluster. [Kubernetes ingress](https://blog.getambassador.io/kubernetes-ingress-nodeport-load-balancers-and-ingress-controllers-6e29f1c44f2d) is a collection of routing rules that govern how external users access services running in a Kubernetes cluster. There are three general approaches for exposing your application:
```

Comment:
How about linking to https://kubernetes.io/docs/concepts/services-networking/ingress/ rather than to https://blog.getambassador.io/kubernetes-ingress-nodeport-load-balancers-and-ingress-controllers-6e29f1c44f2d?

(if the concept page is _not_ good enough to link to, that sounds serious enough to count as a bug).

Reply:
Thanks for the suggestion @sftim. I actually think both links provide value as the blog post adds context to other solutions. I updated the paragraph to hold both links.